components/eam/src/physics/cam/clubb_intr.F90

```diff
@@ -1833,6 +1833,7 @@ subroutine clubb_tend_cam( &
    !  At each CLUBB call, initialize mean momentum  and thermo CLUBB state
    !  from the CAM state
 
+   rvm = 0._r8
```

Comment:
Certainly an initialization of it has no harm. I do wonder if an error was detected with the new debug compilation options. Only 1 to ncol in rvm are used in calculation, and they always have proper values to begin with (from state1%q). If an invalid is reported without the rvm initialization, it seems an over-kill.

Reply:
Array accesses are over `pverp` also in `rvm(1:ncol,1:pverp)`:
```
1944    do i=1,ncol   ! loop over columns
...
2047       do k=1,pverp
...
2060          rvm_in(k)     = real(rvm(i,pverp-k+1), kind = core_rknd)
```
This was needed for `SMS_D_Ln5.ne4_oQU240.F2010.chrysalis_intel.eam-clubb_sp`. Without this initialization, error is 
```
16: forrtl: error (182): floating invalid - possible uninitialized real/complex variable.
16: Image              PC                Routine            Line        Source
16: libpnetcdf.so.3.0  000015555101868C  for__signal_handl     Unknown  Unknown
16: libpthread-2.28.s  000015554E03DB20  Unknown               Unknown  Unknown
16: e3sm.exe           0000000002EECC7C  clubb_intr_mp_clu        2060  clubb_intr.F90
16: e3sm.exe           0000000003546686  physpkg_mp_tphysb        2712  physpkg.F90
16: e3sm.exe           000000000350D8CF  physpkg_mp_phys_r        1150  physpkg.F90
16: libiomp5.so        00001555515583F3  __kmp_invoke_micr     Unknown  Unknown
16: libiomp5.so        00001555514DC273  Unknown               Unknown  Unknown
```
and `rvm = 0._r8` was the minimal initialization to get past this issue.